chapters/prg/python/python-install.md

```diff
@@ -394,6 +385,58 @@ line to your bash configuration located at `~/.bashrc`,
 ```bash
 conda activate ENV3
 ```
+
+#### Activating, Inspecting, and Deactivating Conda
+
+In following the above procedures, only the conda command will be exposed
+at the command line.  To activate our ENV3 environment, run
+
+```bash
+conda activate ENV3
+```
+
+This will augment your current command prompt to use this python environment
+and all of it's dependencies.  Your command prompt will also change to indicate
+that you are running within this environment by prepending `(ENV3)` to your
+prompt.
+
+Note that doing this will also expose any anaconda managed programs to your
+command line (such as openssl and sqlite3).  You can see details on your
+environment by running the command `conda info`.  This shows you many important
+troubleshooting details about your environment, including its name, the path
+where the environment has been configured, and key paths to conda's configuration.
+
+Additionally, you can inspect the current packages installed in the environment
+by running the command `conda list`, which will show both conda packages and python
+packages, their version and build, and the channel (or source) they were installed
+(main, conda-forge, and pypi are the most commonly seen).
+
+When you are done using this conda environment and the software packages it has
+installed, you can deactivate the environment by running
+
+```bash
+conda deactivate
+```
+
+and your shell will remove the conda environment from your current shell's path.
+
+
+### Installing Python via `conda`
+
+Once you have installed conda, you can 
+install Python 3.10 in a virtual environment with conda please use 
+
+```bash
+$ conda create -n ENV3 -c conda-forge python=3.10.2 pip
```

Comment:
Note that this command does two things:

1. Creates a new environment named ENV3
2. Installs in that environment Python 3.10.2, and the latest version of pip that functions with python 3.10.2

This does not activate the environment, a user will need to run the 'conda activate ENV3'